radsat-sk/operation/subsystems/RAntenna.h

```diff
@@ -20,7 +20,7 @@
 #define ANTENNA_I2C_SLAVE_ADDR_REDUNANT (0x32)
 
 /** number of attached Antennas in the system */
-#define ANTENNAS_ON_BOARD 4
+#define ANTENNAS_ON_BOARD 1

```

Comment:
Can you explain the reason for this change? I was busy when discussing the Antenna index.

Reply:
That is because all the antennas communicate over the same I2C line so it can be considered one antenna